wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -591,12 +591,16 @@ extension WireCallCenterV3 {
     }
 
     private func setUpMLSConference(in conversation: ZMConversation) throws {
+        guard let conversationID = conversation.avsIdentifier else {
```

Comment:
just to double check: Is it possible to end up here, and the conversation has no avsIdentifier ?
Could it be we have an ongoing call and the avsIdentifier is not saved properly in the conversation?

Reply:
The existence of the avs identifier is dependent on the existence of the `remoteIdentifier`... which I think is expected to be there, so in theory we should always be able to get the avs id. Kind of annoying that it appears optional, we could try to change this in another PR.